### ISSUE TITLE
[React-Native] React-Navigation Plugin will crash app

### ISSUE THREAD

**Issue by KDederichs** [bugsnag/bugsnag-js#1143]
### Describe the bug
When trying to execute react-native in remote debug mode the app will crash, since synchronous calls are apparently no-op.
Too bad `Bugsnag.getPlugin` seems to fall in that category as well...

### Steps to reproduce
1. Use bugsnag react navigation plugin
2. remote debug

### Environment
* Bugsnag version:
    "@bugsnag/plugin-react-navigation": "^7.5.0",
    "@bugsnag/react-native": "^7.3.5",
* Browser framework version (if any):
    * React: react-native 0.63.3

### Example code snippet 

At the application root level:

```
Bugsnag.start({
  plugins: [new BugsnagPluginReactNavigation()],
});
const { createNavigationContainer } = Bugsnag.getPlugin('reactNavigation');
```

<details><summary>Error messages :</summary>

`_Bugsnag$getPlugin is undefined`
</details>

**Comment by mattdyoung** [comment]
Hi @KDederichs 

This is the same underlying issue as described on https://github.com/bugsnag/bugsnag-js/issues/1112 and we're considering how best we can fix this to cater for when a debugger is attached.

Until a fix is available you'd need to use a workaround similar to the one described on that issue:
https://github.com/bugsnag/bugsnag-js/issues/1112#issuecomment-722970857

**Comment by yousif-bugsnag** [comment]
Fixed in [v7.6.1](https://github.com/bugsnag/bugsnag-js/releases/tag/v7.6.1)
